scripts/build_plugins.sh

```diff
@@ -51,12 +51,21 @@ else
 fi
 p=${BUILD_JOBS:-"${p}"}
 
+# disable parallel xargs where not supported (ie on busybox)
+if [[ ! -x "$(command -v readlink)" || "$(readlink $(which xargs))" == "/bin/busybox" ]]; then
```

Comment:
I'm fine with supporting only Linux case. 
@jcooklin: Are you aware if busybox is supported on Darwin?